src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/domain/BlogPostRepository.java

```diff
@@ -16,7 +16,7 @@
 
     List<BlogPost> findByPublishedDateAfterAndApprovedTrueOrderByPublishedDateAsc(LocalDateTime publishedDate);
 
-    @Query("from BlogPost bp order by bp.publishedDate desc")
+    @Query("from BlogPost bp order by bp.approved desc, bp.publishedDate desc")
```

Comment:
From http://www.postgresql.org/docs/current/static/queries-order.html:

> By default, null values sort as if larger than any non-null value; that is, NULLS FIRST is the default for DESC order, and NULLS LAST otherwise.

I have tested it as well.